.github/workflows/docker-release.yml

```diff
@@ -42,15 +42,8 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        build_preset: ["dev", "lean", "py310", "websocket", "dockerize"]
+        build_preset: ["dev", "lean", "py311", "websocket", "dockerize"]
         platform: ["linux/amd64", "linux/arm64"]
-        exclude:
```

Comment:
is removing this exclude clause deliberate? Actually might be reasonable to remove this from releases, but not from PRs (below)